requirements.txt

```diff
@@ -6,7 +6,7 @@ flake8 == 3.4.1
 psycopg2 == 2.7.3
 PyYAML == 3.11
 requests == 2.7.0
-selenium == 2.53.1
+selenium == 3.4.0
```

Comment:
Since we're bumping up the version, does it make sense to put in checks to ensure that a warning/error is raised if an older version is being used? I'm not sure how version changes are normally handled in projects like these. I'm saying this because this might make the program misbehave for  several contributors/users.

Reply:
If we use pycharm or IDE of similar it usually notifies the change.
We can maybe add a check in __init__ for it (haven't tried this though but worth a try)

Reply:
I don't think there's an __init__.py for this app (at least I couldn't find it), so I think we can leave it for now.